hack/ci-e2e.sh

```diff
@@ -1,10 +1,21 @@
 #!/usr/bin/env bash
 
+# -----------------------------------------------------------------------------
+# Description: This script sets up the environment and runs E2E tests for the
+#              BMO project. It uses either vbmc or sushy-tools based on 
+#              the USE_TOOL environment variable.
+# Usage:       export USE_TOOL="vbmc"  # Or "sushy-tools"
```

Comment:
Could we use more specific and descriptive name for the variable? `BMO_E2E_EMULATOR` or something like that.